pkg/config/template.go

```diff
@@ -0,0 +1,49 @@
+package config
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"strings"
+
+	templateapi "github.com/openshift/api/template/v1"
+	templatescheme "github.com/openshift/client-go/template/clientset/versioned/scheme"
+)
+
+type CiTemplates map[string]*templateapi.Template
```

Comment:
nit: I would prefer to be named just as `Templates`.

Reply:
Do we have some templates that aren't used for CI? If so then I'll rename, otherwise I prefer the more specific name.